reload/init_test.go

```diff
@@ -0,0 +1,17 @@
+package reload_test
+
+import (
+	"testing"
+
+	"github.com/onsi/gomega/format"
+	"github.com/sclevine/spec"
+	"github.com/sclevine/spec/report"
+)
+
+func TestSBOM(t *testing.T) {
```

Comment:
I assume this should be: `TestReload` or similar?

Reply:
You are correct.